src/StaticWebAssetsSdk/Tasks/GenerateStaticWebAssetEndpointsManifest.cs

```diff
@@ -137,6 +139,48 @@ public override bool Execute()
         return !Log.HasLoggedErrors;
     }
 
+    private static void EncodeLinkHeadersIfNeeded(StaticWebAssetEndpoint endpoint)
+    {
+        for (var i = 0; i < endpoint.ResponseHeaders.Length; i++)
+        {
+            ref var header = ref endpoint.ResponseHeaders[i];
+            if (!string.Equals(header.Name, "Link", StringComparison.OrdinalIgnoreCase))
```

Comment:
Of all the headers that might need encoding `Link` is the only one we need to concern ourselves with. Other headers are:
Header | Context of Non-ASCII Value | Encoding Mechanism
-- | -- | --
Content-Disposition | filename parameter (user file names) | RFC 8187 “filename*” parameter encoding
Location / Content-Location | Redirect or resource URIs | Percent-encode; use IDNA (punycode) for domains
Set-Cookie | cookie values; Domain (IDN) | ASCII-only per RFC 6265; use punycode or base64
WWW-Authenticate / Proxy-Authenticate | realm parameter values | RFC 8187 parameter encoding
Warning | warn-text strings | obs-text support in RFC 7230; best to encode
Server / X-Powered-By | Product tokens or comments | obs-text; or percent-encode UTF-8 octets

None of which we are setting as part of SWA.